.circleci/config.yml

```diff
@@ -101,6 +107,10 @@ workflows:
           requires:
           - e2eTestBasic
 
+      - e2eTestMigration:
+          requires:
+          - e2eTestMetrics
```

Comment:
This looks strange to me. Usually the e2e tests just require the build step.

Reply:
Hm, I would say `e2eTestBasic` is required. If helm template is not valid - there is no point to test other things

Reply:
But doesn't that mean the basic test needs to run first? As it is now both tests are run in parallel so you get faster feedback.

IMO the important part is the deployment step requires both e2e tests.